docker-app/qfieldcloud/core/models.py

```diff
@@ -420,7 +419,10 @@ class UserAccount(models.Model):
     @property
     def avatar_url(self):
         if self.avatar_uri:
-            return get_s3_object_url(self.avatar_uri)
+            return reverse_lazy(
```

Comment:
I think there's no need for lazy resolution here (even if it doesn't hurt either)

Reply:
There is a strong bias towards `reverse_lazy` in the codebase so far, so I keep it.